testcafeRunner.js

```diff
@@ -5,6 +5,12 @@ import routes from './app/routes';
 let testcafe;
 let server;
 
+const browserFromArgs = process.argv.slice(2, 3);
+const browserToRun = browserFromArgs.length > 0 ? browserFromArgs : 'chrome:headless';
+
+const testFromArgs = process.argv.slice(3, 4);
+const testsToRun = testFromArgs ? `**/*${testFromArgs}*/ui.test.js` : '**/*ui.test.js';
+
```

Comment:
is this to run headless when on test env? if it is then noice 👍

Reply:
Yes you can pass in the command line arg the browser you want to test and the single test suite you want to run. Don't need a full path just bit of the section you want to test.

Reply:
😍